spec/support/evm_spec_helper.rb

```diff
@@ -89,7 +89,9 @@ def self.clear_instance_variables(instance)
   end
 
   def self.clear_instance_variable(instance, ivar)
-    instance.instance_variable_set(ivar, nil)
+    if instance.instance_variable_defined?(ivar)
```

Comment:
Still a bit confused by this, we're looping through defined instance vars so when would we hit this method with an ivar that isn't defined?  Do the specs pass for you on a mac without this change?  In MiqEnvironment we check `@is_podified.nil?` so wouldn't have expected the difference between setting it to nil and removing it to make a difference

Reply:
When calling `clear_instance_variables`, yes, checking if it is defined first does not make sense, because we are only passing in legit values into here.

When calling `clear_instance_variable` directly, if we were to `remove_instance_variable` for a value that does not exist it would puke